lib/src/components/NxAccordion/__tests__/NxAccordion.test.tsx

```diff
@@ -5,313 +5,212 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import { mount } from 'enzyme';
-import { faChevronCircleDown, faChevronCircleUp } from '@fortawesome/free-solid-svg-icons';
-import { act } from 'react-dom/test-utils';
+import userEvent from '@testing-library/user-event';
 
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
-import NxAccordion, { Props } from '../NxAccordion';
-import NxButton from '../../NxButton/NxButton';
-import NxFontAwesomeIcon from '../../NxFontAwesomeIcon/NxFontAwesomeIcon';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
 
-function createClickEvent() {
-  return new MouseEvent('click', { bubbles: true });
-}
+import NxAccordion from '../NxAccordion';
+import NxButton from '../../NxButton/NxButton';
+import { render } from '@testing-library/react';
 
 describe('NxAccordion', function() {
-  const getShallowComponent = enzymeUtils.getShallowComponent<Props>(NxAccordion, {}),
-      getMountedComponent = enzymeUtils.getMountedComponent<Props>(NxAccordion, {});
+  const quickRender = rtlRender(NxAccordion, {});
+  const renderEl = rtlRenderElement(NxAccordion, {});
 
   it('renders a <details> element with the provided props', function() {
-    const component = getShallowComponent({ id: 'foo', open: true, title: 'test title' });
+    const el = quickRender({ id: 'foo' });
+    expect(el.getByRole('group')).toBe(el.container.firstChild);
+    expect(el.getByRole('group')).toHaveAttribute('id', 'foo');
 
-    expect(component).toMatchSelector('details#foo');
-    expect(component).toHaveProp('open', true);
-    expect(component).toHaveProp('title', 'test title');
+    expect(renderEl({ open: true })).toHaveAttribute('open');
+    expect(renderEl({ title: 'test title' })).toHaveAttribute('title', 'test title');
+
+    expect(quickRender({ open: true }).getByRole('group').tagName).toBe('DETAILS');
```

Comment:
This is over-complicated.  This is what I've been trying to suggest:
```
  it('renders a <details> element with the provided props', function() {
    const el = renderEl({ id: 'foo', open: true })!;
    expect(el.tagName).toBe('DETAILS');
    expect(el).toHaveAttribute('id', 'foo');
    expect(el).toHaveAttribute('open', 'true');
  });

```

Reply:
> This is over-complicated. This is what I've been trying to suggest:
> 
> ```
>   it('renders a <details> element with the provided props', function() {
>     const el = renderEl({ id: 'foo', open: true })!;
>     expect(el.tagName).toBe('DETAILS');
>     expect(el).toHaveAttribute('id', 'foo');
>     expect(el).toHaveAttribute('open', 'true');
>   });
> ```

Okay, that makes sense. d1ec88c521ffb315c0e189bb34198c0fee01e927